distribution/changelog.txt

```diff
@@ -2,6 +2,7 @@
 ------------------------------------------------------------------------
 - Feature: [#13634] Add ability to sell merchandise in random colours.
 - Feature: [#16662] Show a warning message when g2.dat is mismatched.
+- Improvement: [#15358] Park name can now contain up to 128 characters.
```

Comment:
Should there be also a mention that scenario names are up to 256?

Reply:
```suggestion
- Improvement: [#15358] Park and scenario names can now contain up to 128 characters.
```